settings.gradle

```diff
@@ -8,3 +8,14 @@ include 'producers'
 include 'services'
 include 'transformers'
 include 'validators'
+
+dependencyResolutionManagement {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+      url 'https://jitpack.io'
+    }
+    jcenter()
```

Comment:
Is the jcenter() entry needed?  I think the service may not be in use any more.

Reply:
removed